data/joeboadiDev.json

```
@@ -0,0 +1,17 @@
+{
+  "name" : "Joe Boadi",
+  "type" : "personal",
+  "bio" : "Ex-allied health, Tech enthusiast, Software Engineer",
+  "links" : [
+      {
+        "name" : "Twitter",
+        "url" : "https://twitter.com/joeboaadiDev_",
+        "icon" : "FaTwitter"
+      },
+       {
+        "name" : "Github",
+        "url" : "https://github.com/joeboaadiDev",
```

same here also fix your github profile link

```suggestion
        "url" : "https://github.com/joeboadiDev",
```